sites/fast-creator/app/web-components/index.tsx

```diff
@@ -257,3 +261,28 @@ export function renderFormTabs(
         </fast-tabs>
     );
 }
+
+export class HTMLRenderReact extends React.Component {
+    public designRef: React.RefObject<HTMLDivElement>;
```

Comment:
if you assign these both to the product of `React.createRef()` then you can bind the ref without creating new lambda functions every render call:
```ts
 public designRef: React.RefObject<HTMLDivElement> = React.createRef();
// ...
<fast-design-system-provider ref={this.designRef}>
```

Reply:
This scenario seems to require the use of the "ref callback" method. Using the React.createRef() causes errors.
However, I have changed the callback functions to private methods of the class rather than as lambda functions.